manifests/03_operatorsource.crd.yaml

```diff
@@ -1,143 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Shouldn't we also update all of these manifests in the `/deploy/upstream` folder?

https://github.com/operator-framework/operator-marketplace/tree/master/deploy/upstream

Reply:
That is correct. Missed this one. Updating the PR with `/deploy/upstream` udpated.

Reply:
Can you grep for all instances of `OperatorSource` and `CatalogSourceConfig` and make sure that no count is returned?

Reply:
OperatorSource: 
```$ grep -nr 'operatorsource' .
./Makefile:7:OPERATORSOURCE_MOCK_PKG := operatorsource_mocks
Binary file ./build/_output/bin/operator-marketplace-local matches
Binary file ./build/_output/bin/marketplace-operator matches
./pkg/apis/operators/v1/operatorsource_types.go:14:	OpSrcFinalizer = "finalizer.operatorsources.operators.coreos.com"
./pkg/apis/operators/v1/operatorsource_types.go:19:// related to OperatorSource type should be added to operatorsource.go file.
./pkg/migrator/migrator.go:17:	OpsrcCRDName = "operatorsources.operators.coreos.com"
./docs/troubleshooting.md:14:3. [Operators(s) in an OperatorSource fail to show up in the UI](#operators-in-an-operatorsource-fail-to-show-up-in-the-ui) 
./docs/troubleshooting.md:37:`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section](#operatorSource-failing-to-connect-to-the-datastore) and [the following section](#operators-in-an-operatorsource-fail-to-show-up-in-the-ui) of the doc also contains helpful information about debugging an OperatorSource. 
./docs/troubleshooting.md:109:A common reason for an OperatorSource to not reconcile correctly is the use of the wrong [endpoint](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the endpoint is specified as `https://quay.io/cnr`.
./docs/e2e-testing.md:36:    // Create the operatorsource.
./vendor/github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/resolver/resolver.go:77:	// create a map of operatorsourceinfo (subscription+catalogsource data) to the original subscriptions
./README.md:12:The operator manages two CRDs: [OperatorSource](./deploy/upstream/03_operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/upstream/02_catalogsourceconfig.crd.yaml).
./README.md:73:$ kubectl apply -f deploy/upstream/07_upstream_operatorsource.cr.yaml
./README.md:142:[upstream-community-operators]: deploy/upstream/07_upstream_operatorsource.cr.yaml
./README.md:143:[community-operators]: deploy/examples/community.operatorsource.cr.yaml
./manifests/05_role.yaml:29:  - operatorsources
./deploy/upstream/05_role.yaml:19:  - operatorsources
```

CatalogSourceConfigs: 
```$ grep -nr 'catalogsourceconfigs' .
Binary file ./build/_output/bin/operator-marketplace-local matches
Binary file ./build/_output/bin/marketplace-operator matches
./pkg/apis/operators/v2/catalogsourceconfig_types.go:15:	CSCFinalizer = "finalizer.catalogsourceconfigs.operators.coreos.com"
```
For the README updates, operatorsource could be included in this PR: https://github.com/operator-framework/operator-marketplace/pull/312